roles/woodpecker_server/tasks/main.yaml

```
@@ -6,35 +6,29 @@
   tags:
     - woodpecker
 
-- name: Install Woodpecker server
-  community.docker.docker_compose:
-    project_name: woodpecker_server
-    pull: true
-    definition:
-      version: '3'
-      services:
-        woodpecker-server:
-          image: woodpeckerci/woodpecker-server:{{ woodpecker_server_version }}
-          restart: always
-          volumes:
-            - /opt/woodpecker:/var/lib/woodpecker/
-          networks:
-            - web
-            - default
-          environment:
-            - WOODPECKER_OPEN=true
-            - WOODPECKER_HOST={{ woodpecker_server_host }}
-            - WOODPECKER_ORGS={{ woodpecker_server_orgs }}
-            - WOODPECKER_ADMIN={{ woodpecker_server_admins }}
-            - WOODPECKER_GITHUB=true
-            - WOODPECKER_GITHUB_CLIENT={{ woodpecker_server_github_client }}
-            - WOODPECKER_GITHUB_SECRET={{ woodpecker_server_github_secret }}
-            - WOODPECKER_AGENT_SECRET={{ woodpecker_server_agent_secret }}
-            - WOODPECKER_GITHUB_MERGE_REF=false
-      networks:
-        web:
-          external:
-            name: web
+- name: Woodpecker server container
+  community.docker.docker_container:
+    name: woodpecker_server
+    hostname: woodpecker_server
```

```suggestion
    hostname: woodpecker-server
```

Has to be written like this for caddy